test/runtests.jl

```diff
@@ -79,7 +79,6 @@ include("testutils.jl")
 
 for t in tests
     @testset "Test $t" begin
-        Random.seed!(345679)
```

Comment:
The seed is already set above outside of `@testset` and hence it is not necessary to set it here. From the documentation of `@testset`:
```julia
  Before the execution of the body of a @testset, there is an implicit call to
  Random.seed!(seed) where seed is the current seed of the global RNG. Moreover, after the
  execution of the body, the state of the global RNG is restored to what it was before the
  @testset. This is meant to ease reproducibility in case of failure, and to allow seamless
  re-arrangements of @testsets regardless of their side-effect on the global RNG state.
```